tools/api-markdown-documenter/pnpm-lock.yaml

```diff
@@ -989,6 +1007,9 @@ packages:
   character-entities@1.2.4:
     resolution: {integrity: sha512-iBMyeEHxfVnIakwOuDXpVkc54HijNgCyQB2w0VfGQThle6NXn50zU6V/u+LDhxHcDUPojn6Kpga3PTAD8W1bQw==}
 
+  character-entities@2.0.2:
```

Comment:
We ended up with a lot of potentially-dedup-able deps. Not sure what brings in the older versions, maybe worth checking? Although probably best done in a separate PR.